### PR TITLE
extended: bump deployment timeout for test about tagging

### PR DESCRIPTION
**Pull request by kargakis**
@mfojtik for the failure in https://github.com/openshift/origin/pull/10704

### CODE REVIEW

**Comment by kargakis** [comment]
[test]

File: test/extended/testdata/tag-images-deployment.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
   strategy:
     type: Recreate
     recreateParams:
-      timeoutSeconds: 20
+      timeoutSeconds: 600
```
Comment (mfojtik) [inline test/extended/testdata/tag-images-deployment.yaml]: 6 minutes might exceed the timeout of the test

Comment (kargakis) [inline test/extended/testdata/tag-images-deployment.yaml]: Doesn't matter - the test will fail anyway.

Comment (kargakis) [inline test/extended/testdata/tag-images-deployment.yaml]: We just need the failure not to be because of a low timeout:)

**Comment by mfojtik** [comment]
Low risk, fixing flake test [merge]

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**